### ISSUE TITLE
Log entry context parsed incorrectly

### ISSUE THREAD

**Issue by jnoordsij** [getsentry/sentry-php#1649]
### How do you use Sentry?

Sentry SaaS (sentry.io)

### SDK version

4.0.0 (but seems to be appearing on older versions too)

### Steps to reproduce

1. Enable log breadcrumbs
2. Call `Log::info('Test', ['foo', 'bar']);`
3. Trigger an Exception afterwards

### Expected result

The context argument for the log call to be properly visible in the web UI.

### Actual result

Sentry discards the log context with the following message in the web UI displayed:

```
Sentry has identified the following problems for you to fix

Discarded invalid value (1)
Name    `breadcrumbs.value.0.data`
Reason  `expected an object`
Value     `[ 'foo', 'bar' ]`
```

![image](https://github.com/getsentry/sentry-php/assets/6617432/eedefa34-7906-4724-992d-c7e690d97f73)

**Comment by cleptric** [comment]
Fixing this in the PHP SDK, likely here https://github.com/getsentry/sentry-php/blob/master/src/Serializer/Traits/BreadcrumbSeralizerTrait.php#L39-L41

**Comment by cleptric** [comment]
Fixed in [4.3.0](https://github.com/getsentry/sentry-php/releases/tag/4.3.0)
